packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -248,11 +248,23 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
     // Get the latest `encryptedVote` for each (identifier, time).
     const encryptedVoteMap = {};
     for (const ev of encryptedVoteEvents) {
-      encryptedVoteMap[toPriceRequestKey(ev.returnValues.identifier, ev.returnValues.time)] =
-        ev.returnValues.encryptedVote;
+      encryptedVoteMap[
+        // Ancillary data is undefined or null in these "returnValues" so we add a default string of "0x".
+        // If not added, the key will improperly index and the votes will not be found.
+        toPriceRequestKey(
+          ev.returnValues.identifier,
+          ev.returnValues.time,
+          ev.returnValues.ancillaryData || DEFAULT_ANCILLARY_DATA
```

Comment:
Is ev.returnValues.ancillaryData null when the bytes field is empty? I would expect it to be `0x`...